bin/dune

```diff
@@ -1,4 +1,4 @@
 (executable
- (name BsReactIntlPpxBin)
- (public_name bs-react-intl-ppx)
- (libraries bs-react-intl-ppx.lib ppxlib))
+ (name MelangeReactIntlPpxBin)
+ (public_name melange-react-intl-ppx)
```

Comment:
I am not sure this exe should be public, I would only make the ppx library public.

If it has to be public, it should prob need to have a different name than the library.

Reply:
Removed public name:
https://github.com/ahrefs/bs-react-intl-ppx/blob/1cea617f9b80a4f417c05ef5657515ba9cc3b60f/bin/dune#L1-L3